src/scenario.cpp

```diff
@@ -1026,7 +1026,7 @@ scenario::scenario(char * filename, int deflt)
     /* Close scenario element */
     xp_close_element();
     if (xp_is_invalid()) {
-        ERROR("Invalid XML in scenario");
+        ERROR("Invalid XML in scenario.  Check for & and < in strings; replace with &amp; &lt;");
```

Comment:
Maybe:
```
Invalid XML in scenario. See: https://github.com/SIPp/sipp/issues/414
```
Because there could certainly be other reasons than unescaped lt and amp.

Reply:
Here's a commit/branch that adds line numbers in _some_ cases:

https://github.com/SIPp/sipp/commit/a3b6289b2fabf1351d43e050709290b948939144

It's not pretty, but it's fairly non-intrusive.

Please check it this meets your needs.